carta/session.py

```diff
@@ -392,6 +393,24 @@ def append_image(self, path, hdu="", complex=False, expression=ArithmeticExpress
         """
         return Image.new(self, path, hdu, True, complex, expression)
 
+    # @validate(IterableOf(InstanceOf(StokesImage), min_size=2), Boolean())
+    def load_stokes_hypercube(self, stokes_images, append=False):
+        """Open or append a new Stokes hypercube with the selected Stokes images.
+
+        Parameters
+        ----------
+        stokes_images : {0}
+            The list of the paths to the images, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
```

Comment:
This should just be `The list of images.` -- the type should be filled in automatically, and the `StokesImage` class should have its own documentation which explains its parameters.